RocketChat/Rocket.Chat.ReactNative
# [FIX] Enabled and disable the report to bugsnag and analytics

## Proposed changes

Fix Analytics and Crash Reporting opt-out. In addition, fix the flow for both after changing the setting and after restarting the app, disable/enable features on switch changed, and remove the redux.

## Issue(s)	

Enabled and disable analytics and bugsnag isn't work properly when restarting the app and the approach work with bugsnag changed after update the library, which was causing errors.

## How to test or reproduce

- Go to `Security And Privacy` View
- Disable the `Log analytics events` or `Send crash report`
- Then, when you close the app and open it again, your last choice should be the same when reopening the app
- In addition, if you have access to bugsnag or analytics, when you disable anyone of them, there should not report any error or events to their workspace

## Screenshots

## Types of changes




## Checklist



## Further comments
